examples/timerbot.py

```diff
@@ -73,10 +73,11 @@ def unset(bot, update, chat_data):
 
 
 def error(bot, update, error):
-    logger.warning('Update "%s" caused error "%s"' % (update, error))
+    logger.warn('Update "%s" caused error "%s"', update, error)
```

Comment:
Can you change this back to `logger.warning`? `Logger.warn` is deprecated.
https://docs.python.org/3/library/logging.html#logging.Logger.warning